client/src/components/Register.tsx

```diff
@@ -15,7 +15,13 @@ export default function Register() {
   const [error, setError] = useState(false);
 
   const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setUser({ ...user, [event.target.name]: event.target.value });
+    switch (event.target.name) {
+      case "lastName":
+      setUser({ ...user, [event.target.name]: event.target.value ? event.target.value : null });
```

Comment:
Is a switch needed, since this ternary null check would work in every case?

Reply:
Just kidding, obviously not

Reply:
I added it because my console got blasted with errors when I tried to set a non-nullable field to null. Could just be a simple if statement if you think that's better